ruby/examples/app/controllers/user_registration_controller.rb

```diff
@@ -0,0 +1,31 @@
+class UserRegistrationController < ApplicationController
+  def new
+    @user = UserRegistration.new
+  end
+
+  def create
+    request = RegisterUser::Request.new(*user_params)
+    response = RegisterUser.new.register(request)
```

Comment:
Yeah, you don't even need it in statically typed languages. If you look at the kotlin project, I solved this problem by having a class that stands in for that responsibility main would usually have.

Think deject solves this problem quite neatly in Ruby